docs/reference/search/search-vector-tile-api.asciidoc

```diff
@@ -353,6 +358,22 @@ If `true`, the exact number of hits is returned at the cost of some performance.
 If `false`, the response does not include the total number of hits matching the query.
 // end::track_total_hits[]
 
+// tag::with_labels[]
+`with_labels`::
+(Optional, Boolean)
+If true, the hits and aggs layers will contain additional point features representing
+suggested label positions for the original features.
+* `Point` and `MultiPoint` features will have one of the points selected.
+* `Polygon` and `MultiPolygon` features will have a single point generated,
+either the centroid, if it is within the polygon, or another point within the
+polygon selected from the <<geoshape-indexing-approach,sorted triangle-tree>>.
+* `LineString` features will likewise provide a roughly central point selected
+from the <<geoshape-indexing-approach,triangle-tree>>.
+* The aggregation results will provide one central point for each aggregation bucket.
+
```

Comment:
Use `+` instead of empty line so statement below is indented under with_labels option

Reply:
I will add that the label features inherit all the attributes from its corresponding original feature.

Reply:
I had tried that before, but it indented too much (made it part of the last bullet point). However, now I tried again, this time with an extra newline before the `+` and voila! It indented the right amount!

Reply:
I fixed both your suggestions in https://github.com/elastic/elasticsearch/pull/86458/commits/1c69a4aec99dea1891b087c1ed711c730729a0ee